### ISSUE TITLE
Cluster does not allow me to create a clusterbinding operation as the cluster lacks any clusterroles. This is with respect to RBAC.

### ISSUE THREAD

**Issue by yudaykiran** [pires/kubernetes-vagrant-coreos-cluster#262]
Steps to reproduce:
------------------------

I created a cluster using kubeadm to setup a ubuntu vagrant setup and when I run:
```
kubectl get clusterroles
```
I see a bunch of clusterroles displayed which are added by default.

With the kubernetes-vagrant-coreos-cluster "Vagrantfile", I surely get the coreos cluster up, but the cluster does not have any clusterroles defined.

For some of the operations like creating a new clusterbinding requires the user to have a clusterrole of "cluster-admin".

**Comment by pires** [comment]
Everything RBAC-related is created by default so I don't know whta you mean https://kubernetes.io/docs/admin/authorization/rbac/#default-roles-and-role-bindings.

**Comment by yudaykiran** [comment]
If the RBAC-related stuff is created by default, then running 
```
kubectl get clusterroles
```
must be displaying the 'system:' prefixed cluster roles right. But I do not see any clusterroles created.

I have an yaml file that creates an operator. It contains a service, a clusterrole and a clusterbinding, when I try to run it in this setup it give me the following error:

```
Error from server (Forbidden): error when creating "openebs-operator.yaml": clusterroles.rbac.authorization.k8s.io "openebs-maya-operator" is forbidden: attempt to grant extra privileges: [PolicyRule{Resources:["services"], APIGroups:["*"], Verbs:["*"]} PolicyRule{Resources:["pods"], APIGroups:["*"], Verbs:["*"]} PolicyRule{Resources:["deployments"], APIGroups:["*"], Verbs:["*"]} PolicyRule{Resources:["events"], APIGroups:["*"], Verbs:["*"]} PolicyRule{Resources:["persistentvolumes"], APIGroups:["*"], Verbs:["*"]} PolicyRule{Resources:["persistentvolumeclaims"], APIGroups:["*"], Verbs:["*"]} PolicyRule{Resources:["storageclasses"], APIGroups:["storage.k8s.io"], Verbs:["*"]}] user=&{kube-admin  [system:authenticated] map[]} ownerrules=[] ruleResolutionErrors=[]clusterroles.rbac.authorization.k8s.io "cluster-admin" not found
```

The operator gets created without errors in a cluster created by kubeadm. The difference as you said, all the default RBAC stuff was there. I not sure if kubeadm does something.

**Comment by pires** [comment]
I'll have to give it a try but right now I'm overwhelmed with work stuff.

**Comment by yudaykiran** [comment]
Figured out a way to get the default clusterroles, currently testing it out. Will issue a pull request once I am confident that everything is working properly.

**Comment by pires** [comment]
Thanks a lot!

**Comment by yudaykiran** [comment]
Fixed as part of the merge from #263
